src/rqt_graph/dotcode.py

```diff
@@ -225,10 +225,31 @@ def _add_edge(self, edge, dotcode_factory, dotgraph, is_topic=False):
                     _conv(edge.end),
                     label=temp_label,
                     penwidth=penwidth,
-                    color=color)
+                    color=color,
+                    edgetooltip=self._qos_to_string(edge.qos))
             else:
                 dotcode_factory.add_edge_to_graph(
-                    dotgraph, _conv(edge.start), _conv(edge.end), label=edge.label)
+                    dotgraph,
+                    _conv(edge.start),
+                    _conv(edge.end),
+                    label=edge.label,
+                    edgetooltip=self._qos_to_string(edge.qos))
+
+    def _qos_to_string(self, qos):
```

Comment:
`history` and `history_depth` cannot be introspected, that's why the tool is always showing `history=unknown`, `depth=0`. Maybe those policies shouldn't be plot, to avoid confusions.

Reply:
Unfortunate but done in 62ca3505792764cd2952cb7cace7201a87b4162e.